themes/docsy/layouts/partials/footer.html

```diff
@@ -4,9 +4,9 @@
     <img src="/images/logo.svg" alt="">
     <p class="of-footer-main__meta__text">The Operator Framework is an open source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way.</p>
     <ul class="of-link-list of-link-list--secondary">
-      <li class="of-link-list__li"><a class="of-link-list__a" href="#">Operator SDK</a></li>
-      <li class="of-link-list__li"><a class="of-link-list__a" href="#">Operator Lifecycle Manager</a></li>
-      <li class="of-link-list__li"><a class="of-link-list__a" class="of-link of-link--external" href="#">OperatorHub</a></li>
+      <li class="of-link-list__li"><a class="of-link-list__a" href="https://sdk.operatorframework.io/">Operator SDK</a></li>
+      <li class="of-link-list__li"><a class="of-link-list__a" href="https://olm.operatorframework.io/">Operator Lifecycle Manager</a></li>
```

Comment:
Although this is consistent with the other OF sites, it's a little funny that we link to ourselves.